modules/runtime/nuxeo-connect-standalone/src/main/java/org/nuxeo/connect/update/xml/PackageDefinitionImpl.java

```diff
@@ -82,8 +82,15 @@
     /**
      * The target platforms where this package may be installed.
      */
+    @Deprecated
```

Comment:
Could you add a deprecated tag since 11.3 in the javadoc?

Reply:
Forgot to remove it, it should not be deprecated as it will still be used by LTS packages, good catch